src/components/Metropolis/components/Header.tsx

```diff
@@ -131,23 +114,15 @@ const Header = () => {
                             </a>
                             
                             {LINKS.map((link) => {
-                                if (link.style === 'button') {
-                                    return (
-                                        <a href={link.path} key={link.label} target="_blank" className="bg-[#4A5C41]/90 border border-solid border-transparent rounded-full text-center py-1.5 px-[1.15rem] m-0 mt-4 hover:border-[#c7f284] hover:no-underline transition-colors ease-out duration-200">
-                                            <span className="-m-1 text-sm font-semibold inline-block text-[#c7f284]">
-                                                {link.label}
-                                            </span>
-                                        </a>
-                                    );
-                                }
-                                else {
-                                    return (
-                                        <a href={link.path} key={link.label} target="_blank" className='hover:no-underline'>
-                                            <span className="text-sm font-semibold text-white hover:text-[#c7f284] transition-colors ease-in duration-200">{link.label}</span>
-                                        </a>
-                                    );
-                                }
+                                return (
+                                    <a href={link.path} key={link.label} target="_blank" className="text-sm font-semibold py-[0.22rem] text-white transition-colors duration-200 ease-in hover:no-underline hover:text-[#c7f284]">
+                                        {link.label}
+                                    </a>
+                                )
                             })}
+                            <a href={DOCS_URL} target="_blank" className="text-sm font-semibold text-center text-[#c7f284] bg-[#4A5C41]/90 border border-solid border-transparent rounded-full py-[0.6rem] px-[0.93rem] mt-4 transition-colors duration-200 ease-out hover:no-underline hover:text-[#c7f284] hover:border-[#c7f284]">
```

Comment:
these classes are extremely hard to read
would prefer splitting them by section

`[0.6rem] px-[0.93rem]`
and these wont work 
0.93*16 = 14.88px, you can't do .88px